front/abbrevs.tex

```diff
@@ -1,9 +1,79 @@
-% Just write down your abbervations here - or comment-out the command
-
-\abbreviationsAndNotation{
-\begin{tabular}{p{2cm}@{:\quad}l}
-QED & quod erat demonstrandum (``what was to be shown'')\\
-$c$ & the speed of light \\
-$a\pm b$ & the closed interval $\left[a-b,a+b\right]$ \\
-\end{tabular}
+% Use this file to create "glossary entries" for abbreviations and acronyms,
+% and other notation. The entries defined here don't necessarily have to be 
+% used in the thesis (but then you have to decide whether or not to display
+% the unused entries).
+
+% For this file to compile (and the example text in the main/prelims.tex file),
+% the package glossaries-extra is required. It is automatically included unless
+% the noabbrevs class option is used.
+
+% The following will alter the style for typesetting abbreviations when using 
+% the \gls command. Note you can also use multiple styles by categorizing 
+% abbreviations; see the documentation for the glossaries-extras package at:
+% https://ctan.org/pkg/glossaries-extra
+%
+%\setabbreviationstyle[acronym]{long-short-sc}
+%
+% If you're wondering why we're setting the seemingly-redundant "notation 
+% category", that's a hack discussed here:
+% https://tex.stackexchange.com/q/630541/5640 
+
+\newacronym[%
+  category=notation-category,%
+  description=``The Senate and People of Rome'']% The description does not appear anywhere by default
+  {spqr}% the key of the acronym (used with the \gls command for example)
+  {SPQR}% the short form of the acronym
+  {Senātus Populusque Rōmānus}% the long form of the acronym
+
+\newacronym[%
+  category=notation-category,%
+description=A technology used in data storage devices]%
+  {smart}{SMART}{Self-Monitoring, Analysis and Reporting Technology}
+
+\newacronym[%
+  category=notation-category,%
+  description=to build or produce something rather than purchasing it ready-made by others{,} or paying someone else to make it]%
+  {DIY}{DIY}{Do it yourself}
```

Comment:
A minor comment: I think you don't need capital D in Do. According to [Grammarly](https://www.grammarly.com/blog/abbreviations/), when the words of an acronym are presented they only need to be capitalized if they form a proper noun.